Extensions/XEP-0166/XMPPJingleSDP.m

```diff
@@ -540,6 +558,9 @@ - (XMPPIQ *)SDPToXMPP:(NSString *)sdp action:(NSString *)action initiator:(XMPPJ
         if (media_name == nil || [media_name isEqualToString:@""] || [media_name isEqualToString:@"application"]) {
             media_name = @"data";
         }
+        if ([media_name isEqualToString:@"data"]) {
+            continue;
+        }
```

Comment:
@igorcferreira is this @"data" some constant we should put in a variable or parameterise somewhere? I see it being used in multiple places throughout the file

Reply:
@fwroberto Done